### ISSUE TITLE
Couldn't Render Entity

### ISSUE THREAD

**Issue by NillerMedDild** [Cryptic-Mushroom/The-Midnight#313]
When creating a single player world, none of the blocks are visible at first, and the log is spammed with `Couldn't render entity` and `ClassCastException`

Here are the logs, note that I removed some of the json errors to make the file size fit gist
https://gist.github.com/NillerMedDild/3857eb947d0e69afb3d0b04fc6d3bf08

The world never loads completely, however servers work just fine. The issue doesn't seem to be for every new single player world either, it's a bit random.

**Comment by gegy1000** [comment]
This looks like it might be the result of another mod incorrectly exposing a capability. Unfortunately very hard to pinpoint -- it's _some_ living entity holding an inventory

**Comment by NillerMedDild** [comment]
Holding an inventory, could you rephrase/elaborate please?

**Comment by gegy1000** [comment]
An entity that holds items -- it's very vague though. It might be accessible through a GUI or it might be hidden, it depends on the entity :/

**Comment by NillerMedDild** [comment]
Better than nothing, thank you :)
